style.css

```diff
@@ -0,0 +1,46 @@
+@import url("/components/component.css");
+
+html {
+  font-size: 62.5%;
+}
+
+.pos-rel {
+  position: relative;
+}
+
+.pos-abs {
+  position: absolute;
+}
+
+.text-centre {
+  text-align: center;
+}
+
+.remove-deco-white {
+  text-decoration: none;
+  color: var(--white);
+}
+
+body {
+  margin: 0;
```

Comment:
This should be taken care of by the CSS reset in the component library stylesheet.